generator/vulkan/registry.zig

```diff
@@ -65,6 +65,7 @@ pub const Container = struct {
     };
 
     stype: ?[]const u8,
+    extends: ?[]const []const u8,
```

Comment:
These are the contents of `structextends` - so in the future this could be used to typecheck `pNext` chain.